source/common/quic/client_connection_factory_impl.h

```diff
@@ -48,6 +49,8 @@ struct PersistentQuicInfoImpl : public Http::PersistentQuicInfo {
   // This arguably should not be shared across connections but as Envoy doesn't
   // support push promise it's really moot point.
   quic::QuicClientPushPromiseIndex push_promise_index_;
+  // Hard code with real crypto stream for now.
```

Comment:
Please either add a TODO, or change the API docs to make it clear it's downstream only and config validate.

Reply:
This is in listener config now. And the extension config name has "Server" in it, and is also already documented to be used for server crypto stream.

Reply:
Ah sorry, maybe rephrase the comment then?
Hard code with the default crypto stream as there's no pluggable crypto for upstream Envoy?